vtr_flow/benchmarks/system_verilog/f4pga/timer/make_sv_flattened.py

```diff
@@ -0,0 +1,65 @@
+"""
```

Comment:
It seems like you have 3 copies of the make_sv_flattened.py script.  Why not put it in the base f4pga directory instead and just call it from there (with ../../make_sv_flattened.py)?  Or put it in the search path.